fixes #153 add project/relatedProjects array (#178)
open-contracting/infrastructure: docs/reference/schema.md

```

### RelatedProject

> Details of a related project. A project is related when it is involved with the same infrastructure as the current project.

Reviewer: ```suggestion
Details of a project related to the same set of infrastructure assets as the current project.
```
